spec/factories/video.rb

```diff
@@ -0,0 +1,6 @@
+FactoryBot.define do
+  factory :video do
+    title { "MyVideo" }
+    video_url { "www.example.com" }
```

Comment:
Might be worth swapping this over to a proper youtube url, just to make the testing more realistic, in case in the future we feel the need to add some validation

Reply:
Agreed, I'll change this to a TC YouTube video url.